.github/workflows/hardware-smoke.yml

```diff
@@ -105,7 +103,7 @@ jobs:
           # Run tests tagged with "smoke"
           ./scripts/twister -i --retry-failed 3 \
             -p $SMC_BOARD --device-testing \
-            --hardware-map /opt/tenstorrent/twister/hw-map.yml --west-flash \
+            --hardware-map /opt/tenstorrent/twister/hw-map-vuart.yml --west-flash \
```

Comment:
Once this has soaked for a couple of days, we should be able to update CI systems and make this the default. I think it will require some additional board-level default configuration, but should be a relatively minor change, like flipping a switch.